pages/operators/validators/join-the-proof-of-authority.md

```diff
@@ -0,0 +1,18 @@
+# Join the Proof of Authority with your node
+Similar to the XRPL mainnet, the Devnet runs in a Proof of Authority consensus mechanism. In order to start signing for new blocks and participate in the network consensus, the current validators need to accept your node as a new trusted validator. This democratic process requires the approval of the majority of the current validators.
```

Comment:
The consensus mechanism of XRPL is not Proof of Authority.
The one of XRPL is sometimes called PoA, but it is Proof of Association.

Reply:
The text was meant to say that mainnet consensus has similarities with proof of authority, despite being different. In order to avoid confusions, I have removed the mention to XRPL